elasticsearch/Dockerfile

```diff
@@ -7,24 +7,23 @@ RUN swupd update $swupd_args \
 	&& swupd bundle-add elasticsearch $swupd_args \
 	&& rm -rf /var/lib/swupd/*
 
-ENV PATH /usr/share/elasticsearch/bin:$PATH
+ENV PATH="/usr/share/elasticsearch/bin:${PATH}"
 
 WORKDIR /usr/share/elasticsearch
 
 RUN set -ex \
 	&& for path in \
-		./data \
-		./logs \
+		/var/data/elasticsearch \
+		/var/log/elasticsearch \
 		./config \
-		./config/scripts \
 	; do \
 		mkdir -p "$path"; \
```

Comment:
So this still makes the config directory in /usr/share/elasticsearch. Would it be possible to run elasticsearch with an alternate config path? Otherwise this is probably as good as it gets I guess.

Reply:
I got your points, will try.